task_handler_layer/task_gate/src/gate_detection.cpp

```diff
@@ -37,7 +37,7 @@ void callback(task_gate::gateConfig &config, uint32_t level)
   t2max = config.t2max_param;
   t3min = config.t3min_param;
   t3max = config.t3max_param;
-  ROS_INFO("Gate_Reconfigure Request:New params : %d %d %d %d %d %d ", t1min, t1max, t2min, t2max, t3min, t3max);
+  ROS_INFO("Reconfigure Request : New parameters : %d %d %d %d %d %d ", t1min, t1max, t2min, t2max, t3min, t3max);
```

Comment:
Keep `Gate_Reconfigure` as it helps identify which node created the rosinfo log.